build.assets/Dockerfile-connect

```
@@ -0,0 +1,47 @@
+# This Dockerfile makes the "build box connect" the container used
+# to build the Teleport Connect.
+#
+# This image is base on the node image, which is based on Debian.
+# Using it as a image allows us to link agains the same version of
+# glibc as Node.js.
+#
+# Check the README to learn how to safely introduce changes to Dockerfiles.
+
+## BUILDBOX-CONNECT ###################################################################
+
+ARG NODE_VERSION
+FROM node:${NODE_VERSION} AS buildbox
+
+COPY locale.gen /etc/locale.gen
+COPY profile /etc/profile
+ENV LANGUAGE="en_US.UTF-8" \
+    LANG="en_US.UTF-8" \
+    LC_ALL="en_US.UTF-8" \
+    LC_CTYPE="en_US.UTF-8" \
+    DEBIAN_FRONTEND="noninteractive"
+
+# Install packages.
+RUN apt-get -y update && \
+    apt-get -y install software-properties-common && \
```

Why does this need a separate step?

nit: Use same options as below?

```suggestion
    apt-get install -q -y --no-install-recommends software-properties-common && \
```

---

I decided to remove it. We don't need that package now and because we moved from Ubuntu to Debian I don't think this package will be ever needed.